Dockerfile

```diff
@@ -1,56 +1,26 @@
 FROM registry.fedoraproject.org/fedora:latest
 
-RUN dnf -y update && dnf install -y make git golang golang-github-cpuguy83-md2man \
-	# storage deps
-	btrfs-progs-devel \
-	device-mapper-devel \
-	# gpgme bindings deps
-	libassuan-devel gpgme-devel \
-	gnupg \
-	# htpasswd for system tests
-	httpd-tools \
-	# OpenShift deps
```

Comment:
Any chance to preserve these comments?

Reply:
Actually, I'm not a big fan of maintaining a package list inside the Dockerfile (it often becomes unwieldy).  Let me investigate an alternative that will also allow keeping these comments.

Reply:
Nice.